package.json

```diff
@@ -12,22 +12,21 @@
     "console": "truffle console",
     "coverage": "solidity-coverage",
     "flatten": "node ./scripts/flatten.js 2>&1",
-    "flatten:witnet": "npm run clean && npm run flatten:witnet:libs && npm run flatten:witnet:proxy && npm run flatten:witnet:board",
-    "flatten:witnet:libs": "npm run flatten contracts/libs/Witnet.sol && npm run flatten contracts/libs/CBOR.sol",
-    "flatten:witnet:proxy": "npm run flatten contracts/WitnetProxy.sol",
-    "flatten:witnet:board": "npm run flatten:witnet:board:trustable && npm run flatten:witnet:board:trustless",
-    "flatten:witnet:board:trustable": "npm run flatten contracts/impls/trustable/",
-    "flatten:witnet:board:trustless": "npm run flatten contracts/impls/trustless/",
+    "flatten:witnet": "npm run clean && npm run flatten:witnet:libs && npm run flatten:witnet:proxy && npm run flatten:witnet:boards",
+    "flatten:witnet:libs": "npm run flatten contracts/libs/WitnetDecoderLib.sol && npm run flatten contracts/libs/WitnetParserLib.sol",
+    "flatten:witnet:proxy": "npm run flatten contracts/impls/WitnetProxy.sol",
+    "flatten:witnet:boards": "npm run flatten:witnet:boards:trustable && npm run flatten:witnet:boards:trustless",
```

Comment:
There is no trustless folder, so this command fails

Reply:
Right, as empty folders are not commited to github